packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -76,10 +77,15 @@ def test_if_uiconfig_is_available(cluster):
 
 
 def test_if_dcos_history_service_is_up(cluster):
-    r = cluster.get('/dcos-history-service/ping')
 
-    assert r.status_code == 200
-    assert 'pong' == r.text
+    @retry(stop_max_delay=20000, wait_fixed=500)
```

Comment:
this functionality is already covered when we make the cluster fixture: https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/conftest.py#L87
which does:
https://github.com/dcos/dcos/blob/master/test_util/cluster_api.py#L58
Thus, I see no reason to modify this test